util/chplenv/chpl_locale_model.py

```diff
@@ -2,12 +2,17 @@
 import sys
 
 import overrides
-from utils import memoize
+from utils import memoize, error
 
 
 @memoize
 def get():
     locale_model_val = overrides.get('CHPL_LOCALE_MODEL', 'flat')
+
+    if locale_model_val != 'flat' and locale_model_val != 'gpu':
```

Comment:
Nit, but in the past for something like this I'll write it like this:

`if locale_model_val not in ['flat', 'gpu']`

Just to make it easier to add new entries if/when that happens.